Casks/fluor.rb

```diff
@@ -8,6 +8,11 @@
   name "Fluor"
   homepage "https://fluorapp.net/"
 
+  livecheck do
+    url :homepage
+    regex(/%20(\d+(?:\.\d+)*)\.dmg/i)
```

Comment:
```suggestion
    regex(/href=.*?Fluor(?:%20|[._-])v?(\d+(?:\.\d+)+)\.dmg/i)
```

Among the usual changes, this matches the encoded space (`%20`) after the software name but also allows the usual delimiters (`[._-]`), in case it changes in the future.